test/postman/check-keycloak-access-token-lifespan-change-persisted.json

```diff
@@ -0,0 +1,227 @@
+{
+	"info": {
+		"_postman_id": "152d7ee3-dc69-40f0-a381-ebe16d9e62b3",
+		"name": "check-keycloak-access-token-lifespan-change-persisted",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Generate Keycloak admin access token",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "bcedf8cbe-1d10-4dce-991c-678762ee8103",
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"var jsonData = pm.response.json();",
+							"",
+							"pm.test(\"Token is bearer token\", function () {",
+							"    pm.expect(jsonData.token_type).to.eql(\"bearer\");",
+							"});",
+							"",
+							"pm.test(\"Response contains access token\", function () {",
+							"    pm.expect(\"access_token\" in jsonData).to.be.true;",
+							"});",
+							"",
+							"var tokens = JSON.parse(responseBody); ",
+							"pm.environment.set(\"kcAdminAccessToken\", tokens.access_token);",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/x-www-form-urlencoded"
+					}
+				],
+				"body": {
+					"mode": "urlencoded",
+					"urlencoded": [
+						{
+							"key": "client_id",
+							"value": "{{identityServiceClientId}}",
+							"type": "text"
+						},
+						{
+							"key": "grant_type",
+							"value": "password",
+							"type": "text"
+						},
+						{
+							"key": "username",
+							"value": "{{adminUserUsername}}",
+							"type": "text"
+						},
+						{
+							"key": "password",
+							"value": "{{adminUserPassword}}",
+							"type": "text"
+						}
+					]
+				},
+				"url": {
+					"raw": "{{protocol}}://{{identityServiceUrl}}/realms/{{identityServiceRealm}}/protocol/openid-connect/token",
+					"protocol": "{{protocol}}",
+					"host": [
+						"{{identityServiceUrl}}"
+					],
+					"path": [
+						"realms",
+						"{{identityServiceRealm}}",
+						"protocol",
+						"openid-connect",
+						"token"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "get access token lifespan",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "f0d890fb-91df-4834-81d8-eced6bf4145a",
+						"exec": [
+							"var jsonData = pm.response.json();",
+							"",
+							"console.log(\"---------------------------------------------\");",
+							"console.log(jsonData.accessTokenLifespan);",
+							"",
+							"",
+							"",
+							"pm.test(\"Status code is 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"",
+							"",
+							"pm.test(\"value of access token lifespan to be 7200\", function () {",
+							"    pm.expect(jsonData.accessTokenLifespan).to.equal(7200);",
```

Comment:
Can this hardcoded value `7200` be made a variable and this variable used here instead? That would make the collection more easily maintainable.
The same change needs to be applied to `change-keycloak-access-token-lifespan-collection.json` but for some reason I can't see its full diff in this PR.

Reply:
Has this `7200` been removed from this file as discussed? I can't see this change but it could be that github code review tool is having its "moment".

Reply:
Yes, it is now stored in a variable called `expectedAccessTokenLifespan`, however in retrospect, i only changed it in the `change-keycloak-access-token-lifespan-collection.json` and the get request is actually duplicated in anther collection `check-keycloak-access-token-lifespan-change-persisted.json` so I have changed it in there too now. I am not sure if you know a more efficient way than two have two of the same tests in different collections?

Reply:
I'm afraid in this scenario you can't do anything much more elegant (1st GET is to verify it's been applied, 2nd copy is to verify it after node restart, if I got it right). It's good as it is right now, thanks.